tests/simulator/test_advance.py

```
@@ -214,189 +251,180 @@ def vthz(*xyz):
+
+                slice1 = boxm.select(pd1.dataset, box_pd1)
+                slice2 = boxm.select(pd2.dataset, box_pd2)
+
+                try:
+                    # empirical max absolute observed 5.2e-15
+                    # https://hephaistos.lpp.polytechnique.fr/teamcity/buildConfiguration/Phare_Phare_BuildGithubPrClang/78544
+                    # seems correct considering ghosts are filled with schedules
+                    # involving linear/spatial interpolations and so on where
+                    # rounding errors may occur.... setting atol to 5.5e-15
+                    assert_fp_any_all_close(slice1, slice2, atol=5.5e-15, rtol=0)
+                    checks += 1
```

Ruff (0.14.4)

Undefined name `checks`

(F821)